.github/workflows/ci.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - name: Run tests
         run: |
           gem install bundler
-          rm Gemfile.lock
```

Comment:
Sorry I forgot to point that out, we shouldn't have removed it. Removing the lock in the context of CI ensures that the gem doesn't break whenever a depenencies gets updated

Reply:
Or dependabot can be added, that's less flaky too (because removing Gemfile.lock makes it fail randomly whenever a build happens).